### ISSUE TITLE
Inventory Accruals

### ISSUE THREAD

**Issue by koushik-ac** [beezlabs-org/automation-dev#183]

**Comment by Dean13b** [comment]
Received Budget approval

**Comment by Dean13b** [comment]
Call to be scheduled for the PDD approval

**Comment by koushik-ac** [comment]
We have received soft Approval for the PDD. A change has been suggested in the process which the team will start working on from today. ETA to be given by EOD.

**Comment by koushik-ac** [comment]
Test data Dependency to proceed with Testing

**Comment by Dean13b** [comment]
Test data/scenarios to be identified

**Comment by koushik-ac** [comment]
Test Data/ Scenarios has not yet been provided to us. As agreed, unless we get proper test data and test scenarios, we cannot move to UAT.

**Comment by koushik-ac** [comment]
Test Data received from end users today. We have to review it and start testing the automation.

**Comment by koushik-ac** [comment]
@anandamurugan-s is testing the SQ00 bot and looking into the failure reason. @koushik-ac to analyze the test data provided.

**Comment by anandamurugan-s** [comment]
The following issues were addressed and resolved:

1. The problem of displaying "0,00" in VM for Excel sheet values of 0 was fixed in the MB5S bot.
2. The sq00 bot was fixed to address issues related to empty (PO) or (FI) file signatures.
3. An Excel manipulation bot was implemented to handle the empty signature problem in PO or FI files and address invalid URL issues in the (DMS).

**Comment by anandamurugan-s** [comment]
<img width="746" alt="image" src="https://github.com/beezlabs-org/automation-dev/assets/151733184/d6d2145e-58b9-4621-912a-e9e82a96594d">

Need to address the above issue in share point bot

**Comment by Dean13b** [comment]
Testing in progress. UAT call planned for 4th January(subjective to change as per client's availability)

**Comment by koushik-ac** [comment]
UAT Call has been scheduled for 3rd Jan. We are yet receive any update about the Client's availability for the call.

**Comment by anandamurugan-s** [comment]
I am starting to check the PO Extraction bot, and it seems that the bot is failing to update the PO number in the output schema.

**Comment by anandamurugan-s** [comment]
Completed PO and FI filepath checks in the SQ00 bot.
Modified the email rule logic as per the latest requirement.
Optimized the output schema for the FBICR3 marking bot based on the GRIR status.

**Comment by koushik-ac** [comment]
1st UAT has been completed on 03/01/2023. We have showcased the demo for all the test data we have received. The output results were shared with the team to validate them and provide their comments. In the meantime, we have requested additional test data to test different scenarios as suggested by Juan. Waiting for an update from them.

**Comment by Dean13b** [comment]
Tulip upgrade is a dependency for go-live

**Comment by koushik-ac** [comment]
Yes Priyanka. As per the schedule, we completed 1st UAT on 3rd January itself with all the available test data. We have raised this with the team almost one month back. But we are yet to receive any update from them. So, we have pushed the UAT Hand over date once the test data is only available.

**Comment by koushik-ac** [comment]
UAT Checkpoint call has been setup for Thursday 11/01/2024. We will check with the end users about the test data availability and further testing.

**Comment by koushik-ac** [comment]
UAT has been completed and the files were sent for review to end users. They have asked us to onboard all the different users to Tulip and also give necessary access to Dev Cockpit. They have highlighted a formatting issue. I shall be looking into it today.

**Comment by mbharathm** [comment]
@koushik-ac , please provide visibility on the issue raised with the numbering format on the latest email.



![Image](https://github.com/beezlabs-org/automation-dev/assets/64949233/9ec03a04-241e-4c99-9755-d943c41c50b4)

**Comment by koushik-ac** [comment]
The users have asked for a change in the way data is downloaded. The changes were implemented and tested from our side. We are awaiting further testing from the end users for these changes.

**Comment by mbharathm** [comment]
@koushik-ac , I don't see an email sent on the fix confirmation. Was this sent in any other email thread? Please confirm.

**Comment by koushik-ac** [comment]
Yes Bharath. It was sent. The end user has confirmed that he shall be testing the solution this week and next week as well.

**Comment by mbharathm** [comment]
@koushik-ac , can you confirm if this is raised/approved with security team? or is it already existing under IPA_RTR.

![Image](https://github.com/beezlabs-org/automation-dev/assets/64949233/692d88a8-b098-4dcd-b46c-69d48534ec0a)

**Comment by Dean13b** [comment]
Has dependency with Tulip upgrade

**Comment by koushik-ac** [comment]
@mbharathm - This was already raised by Juan directly for getting the necessary access. Currently, IPA_RTR has access to FBICR3 and MB5S.
